scanner/updater/export.go

```diff
@@ -56,42 +57,51 @@ func Export(ctx context.Context, outputDir string) error {
 		}
 	}()
 
+	var opts [][]updates.ManagerOption
+
+	// Manual CVEs.
 	manualSet, err := manual.UpdaterSet(ctx, nil)
 	if err != nil {
 		return err
 	}
-
 	outOfTree := append([]driver.Updater{}, manualSet.Updaters()...)
+	opts = append(opts, []updates.ManagerOption{updates.WithOutOfTree(outOfTree)})
```

Comment:
are we able to just do 

```
opts = append(opts, []updates.ManagerOption{updates.WithOutOfTree(manualSet.Updaters())})
```

Reply:
I assumed there was a reason, and haven't checked, so I left it alone.

Reply:
I don't think there really was a reason, but we can keep it as-is